pkg/rollup/config.go

```diff
@@ -0,0 +1,82 @@
+package rollup
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/jackc/pgx/v4"
+
+	"github.com/timescale/promscale/pkg/internal/day"
+	"github.com/timescale/promscale/pkg/log"
+)
+
+type Config struct {
+	Enable               *bool `yaml:"enabled,omitempty"`
```

Comment:
NIT: let's use the same as the config key 

```suggestion
	Enabled               *bool `yaml:"enabled,omitempty"`
```